src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.tsx

```diff
@@ -167,7 +163,7 @@ const TranscriptSummary = () => {
   const { gene, transcript } = data;
   const defaultProductGeneratingContext = transcript.product_generating_contexts.find(
     (entry) => entry.default
-  ) as ProductGeneratingContext;
+  ) as Transcript['product_generating_contexts'][number];
```

Comment:
Would it be better if we define `Transcript['product_generating_contexts'][number]` separately and refer it here?
Like `type ProductGeneratingContext = Transcript['product_generating_contexts'][number]`

Reply:
✅